test/cookies.jl

```diff
@@ -8,123 +8,129 @@
-        (Dict(["Set-Cookie"=> "special-1=a z"]),  [HTTP.Cookie("special-1", "a z")]),
-        (Dict(["Set-Cookie"=> "special-2=\" z\""]), [HTTP.Cookie("special-2", " z")]),
-        (Dict(["Set-Cookie"=> "special-3=\"a \""]), [HTTP.Cookie("special-3", "a ")]),
-        (Dict(["Set-Cookie"=> "special-4=\" \""]),  [HTTP.Cookie("special-4", " ")]),
-        (Dict(["Set-Cookie"=> "special-5=a,z"]),  [HTTP.Cookie("special-5", "a,z")]),
-        (Dict(["Set-Cookie"=> "special-6=\",z\""]), [HTTP.Cookie("special-6", ",z")]),
-        (Dict(["Set-Cookie"=> "special-7=a,"]),   [HTTP.Cookie("special-7", "a,")]),
-        (Dict(["Set-Cookie"=> "special-8=\",\""]),  [HTTP.Cookie("special-8", ",")]),
-    ]
-
-    for (h, c) in cookietests
-        @test HTTP.Cookies.readsetcookies("", [Dict(h)["Set-Cookie"]]) == c
-    end
-end
-
-@testset "SetCookieDoubleQuotes" begin
-    cookiestrings = [
-        "quoted0=none; max-age=30",
-        "quoted1=\"cookieValue\"; max-age=31",
-        "quoted2=cookieAV; max-age=\"32\"",
-        "quoted3=\"both\"; max-age=\"33\"",
-    ]
-    want = [
-        HTTP.Cookie("quoted0", "none", maxage=30),
-        HTTP.Cookie("quoted1", "cookieValue", maxage=31),
-        HTTP.Cookie("quoted2", "cookieAV"),
-        HTTP.Cookie("quoted3", "both"),
-    ]
-    @test all(HTTP.Cookies.readsetcookies("", cookiestrings) .== want)
-end
-
-@testset "Cookie sanitize value" begin
-    values = Dict(
-        "foo" => "foo",
-        "foo;bar" => "foobar",
-        "foo\\bar" => "foobar",
-        "foo\"bar" => "foobar",
-        String(UInt8[0x00, 0x7e, 0x7f, 0x80]) => String(UInt8[0x7e]),
-        "\"withquotes\"" => "withquotes",
-        "a z" => "a z",
-        " z" => "\" z\"",
-        "a " => "\"a \"",
-    )
-    for (k, v) in values
-        @test HTTP.Cookies.sanitizeCookieValue(k) == v
-    end
-end
-
-@testset "Cookie sanitize path" begin
-    paths = Dict(
-        "/path" => "/path",
-        "/path with space/" => "/path with space/",
-        "/just;no;semicolon\0orstuff/" => "/justnosemicolonorstuff/",
-    )
-    for (k, v) in paths
-        @test HTTP.Cookies.sanitizeCookiePath(k) == v
-    end
-end
-
-@testset "HTTP.readcookies" begin
-    testcookies = [
-        (Dict("Cookie" => "Cookie-1=v\$1; c2=v2"), "", [HTTP.Cookie("Cookie-1", "v\$1"), HTTP.Cookie("c2", "v2")]),
-        (Dict("Cookie" => "Cookie-1=v\$1; c2=v2"), "c2", [HTTP.Cookie("c2", "v2")]),
-        (Dict("Cookie" => "Cookie-1=v\$1; c2=v2"), "", [HTTP.Cookie("Cookie-1", "v\$1"), HTTP.Cookie("c2", "v2")]),
-        (Dict("Cookie" => "Cookie-1=v\$1; c2=v2"), "c2", [HTTP.Cookie("c2", "v2")]),
-        (Dict("Cookie" => "Cookie-1=\"v\$1\"; c2=\"v2\""), "", [HTTP.Cookie("Cookie-1", "v\$1"), HTTP.Cookie("c2", "v2")]),
-    ]
-    for (h, filter, cookies) in testcookies
-        @test HTTP.Cookies.readcookies(h, filter) == cookies
-    end
-end
-end; # @testset "Cookies"
+		(HTTP.Cookie("cookie-1", "v\$1"), "cookie-1=v\$1"),
```

Comment:
why the indentation change here?

Reply:
Looks like this happened accidentally, also this file is using tabs. Fixing this now.